go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
 	github.com/pkg/errors v0.9.1
 	github.com/sclevine/spec v1.4.0
+	github.com/shurcooL/go v0.0.0-20180423040247-9e1955d9fb6e
```

Comment:
What is this used for?

Reply:
not directly used

`go mod graph` gives this as the output
```
...
github.com/sourcegraph/go-diff@v0.5.3 github.com/gogo/protobuf@v1.1.1
github.com/sourcegraph/go-diff@v0.5.3 github.com/shurcooL/go@v0.0.0-20180423040247-9e1955d9fb6e
github.com/sourcegraph/go-diff@v0.5.3 github.com/shurcooL/go-goon@v0.0.0-20170922171312-37c2f522c041
...
```
and `go mod why github.com/shurcooL/go` gives
```
go: finding module for package github.com/shurcooL/go
# github.com/shurcooL/go
(main module does not need package github.com/shurcooL/go)
```
However `go mod tidy` does not take it out. And if I manually remove it from go.mod and go.sum and run tests, it comes right back and so, I left it in

Reply:
happy to take it out manually. should I?

Reply:
It looks like it got accidentally pulled in here. https://github.com/pivotal/kpack/blob/693ee761d2df7e0a41ed15e64b349e15a892f82c/cmd/rebase/main.go#L15

Reply:
I will create a PR right now to remove it.

Reply:
https://github.com/pivotal/kpack/pull/593

Reply:
rebased from master. resolved with #593